client/packages/programs/src/JsonForms/common/components/Number.tsx

```diff
@@ -47,8 +48,12 @@ const UIComponent = (props: ControlProps) => {
       labelWidthPercentage={FORM_LABEL_WIDTH}
       inputAlignment="start"
       Input={
-        schema.minimum !== undefined ? (
-          <PositiveNumberInput {...inputProps} min={schema.minimum} />
```

Comment:
This is a PostiitveNumberInput... so minimum should always be 0 unless defined otherwise.....

Reply:
Should really have min and max in NumericTextInput similar to what is done for the PositiveNumberInput... Then we don't need a distinction here...Because currently we also don't handle negative minimum values...

@mark-prins can we build this into NumericTextInput or can we create a new RangedNumericTextInput?

Reply:
yes, I agree - would be good to just extend the NumericTextInput. The PositiveNumber and NegativeNumber inputs are just shorthand - not used in many places, so may as well just put optional params on the NumericTextInput